### ISSUE TITLE
Goron City pot isn't providing the Kokiri Sword

### ISSUE THREAD

**Issue by GonthorianDX** [AmazingAmpharos/OoT-Randomizer#210]
So I got the following seed and settings:

>OoT Randomizer Version v3.0  -  Seed: M8IOTFE0H2
Settings (V42R3RANF282AKAJ2B):

I found a gossip stone that hinted me that the Kokiri Sword is in Goron City, I checked the two chests in the maze but neither contains the sword so I looked it up in the spoiler log.

>Goron City Pot Freestanding PoH: Kokiri Sword

I threw well over 10 bombs into the pot but it either gives me nothing at all or 3 green rupees.
It seems this pot is bugged for me, luckily it isn't a mandatory item as I can use sticks for when I have no alternative.
Please fix this issue in the next update, thank you.

I included a savestate for RetroArch so you can see it for yourself.
[OoT_V42R3RANF282AKAJ2B_M8IOTFE0H2-comp.zip](https://github.com/AmazingAmpharos/OoT-Randomizer/files/2680251/OoT_V42R3RANF282AKAJ2B_M8IOTFE0H2-comp.zip)

**Comment by Physonian** [comment]
You have to throw the bomb into the pot at a certain time. Look it up on the internet. I am sure that it is not bugged

**Comment by GonthorianDX** [comment]
It came down to really bad luck I think, after trying some more, the heart piece finally came out.

It would be cool if you could modify the game code so you always get the item first try.

**Comment by phixtyseven** [comment]
It's apparently a matter of timing, so there are no plans of changing the behavior.

**Comment by flagrama** [comment]
Yeah, it is 100% skill-based. You'll be able to remove it (and any location) in a future version if you really can't do it, but there is no chance of it being changed to first try like Dampe's Gravedigging.
